custom_components/hubitat/climate.py

```diff
@@ -236,6 +236,13 @@ def temperature_unit(self) -> str:
             return TEMP_FAHRENHEIT
         return TEMP_CELSIUS
 
+    @property
+    def precision(self) -> Optional[float]:
+        """Return current temperature precision when Farenheit and thermostat reports decimals."""
+        if self.temperature_unit == TEMP_FAHRENHEIT and "." in str(self.current_temperature):
+            return 0.1
+        return None
```

Comment:
Rather than returning None, this should return the precision for non-Fahrenheit values. From playing with the Virtual Thermostat, it seems like that might also be `PRECISION_TENTHS` (in which case this whole method could just return `PRECISION_TENTHS`). I haven't used it much, though, and if Hubitat only does use whole numbers for thermostats in Celsius mode, this should return `PRECISION_WHOLE`.